packages/nx/src/daemon/client/client.ts

```diff
@@ -608,6 +608,10 @@ export class DaemonClient {
       await this.sendMessageToDaemon({ type: FORCE_SHUTDOWN });
       await waitForDaemonToExitAndCleanupProcessJson();
     } catch (err) {
+      if (err.code === 'ENOENT') {
```

Comment:
This seems like the wrong kind of error to be expecting in this function. sending a message to a daemon which does not exist, should throw a special error... which we can catch here.